be/src/common/config.h

```diff
@@ -872,6 +872,8 @@ CONF_mBool(enable_new_load_scan_node, "false");
 // Temp config. True to use new file scanner. Will remove after fully test.
 CONF_mBool(enable_new_file_scanner, "false");
 
+CONF_mString(be_node_type, "be");
```

Comment:
how about `mixed`, `compute`, etc.
And I think this is not a modifiable config, so it should be `CONF_String`